### PR TITLE
[unmanaged_ami] Instructions on launching the unmanaged AMI

### PR DESCRIPTION
**Pull request by kouzant**

### CODE REVIEW

File: docs/getting_started/installation_guide/platforms/aws-image.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.. _aws-image:
+
+Amazon Machine Images
+=====================
+
+Hopsworks 1.4.1 is available as a community AMI ready to be launched. You can use it to
```
Comment (tkakantousis) [inline docs/getting_started/installation_guide/platforms/aws-image.rst:6]: No need to mention the version here as the docs are already versioned and we will forget to update with subsequent releases

File: docs/getting_started/installation_guide/platforms/aws-image.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+easily create a single-node Hopsworks instance potentially with **dedicated GPU**
```
Comment (tkakantousis) [inline docs/getting_started/installation_guide/platforms/aws-image.rst:7]: I would omit `potentially` or replace it with a word like `optionally`

File: docs/getting_started/installation_guide/platforms/aws-image.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+for accelerated Deep Learning.
+
+Launch instance
+------------------
+To launch an EC2 instance with Hopsworks install you will need an AWS account.
```
Comment (tkakantousis) [inline docs/getting_started/installation_guide/platforms/aws-image.rst:12]:
```suggestion
To launch an EC2 instance with Hopsworks install, you will need an AWS account.
```

File: docs/getting_started/installation_guide/platforms/aws-image.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Open Amazon EC2 console https://console.aws.amazon.com/ec2
+2. Click **Launch instance**
+3. On the **Choose an Amazon Machine Image (AMI)** page, choose **Community AMIs** on the left
+4. Search for ``LC-hopsworks-1.4.1`` and select the latest image
+5. On the **Choose an Instance Type** page select the hardware configuration for you instance. For a smooth experience we recommend at **minimum** ``t2.2xlarge`` instance type.
+6. Continue with configuring **Instance details** Make sure that:
+
+   - The VPC you select has enabled **Private DNS hostnames** By default when you create a new VPC, it's not enabled. To enabled it follow this guide here_
```
Comment (tkakantousis) [inline docs/getting_started/installation_guide/platforms/aws-image.rst:21]:
```suggestion
   - The VPC you select has **Private DNS hostnames** enabled. By default when you create a new VPC, it's not enabled. To enabled it follow this guide here_
```

**Review by tkakantousis — CHANGES_REQUESTED** [review]

**Comment by kouzant** [comment]
@tkakantousis Fixed your comments

**Review by tkakantousis — APPROVED** [review]

**Comment by tkakantousis** [comment]
@kouzant will this be merged in branch 1.4 as well?

**Comment by kouzant** [comment]
> @kouzant will this be merged in branch 1.4 as well?

Yes, since this PR is OK I'll open the same for 1.4

**Pull request merged**